packages/datadog-plugin-http/src/client.js

```diff
@@ -227,7 +227,7 @@ function hasAmazonSignature (options) {
       return true
     }
 
-    if ([].concat(headers['authorization']).some(startsWith('AWS4-HMAC-SHA256'))) {
+    if ([].concat(headers.authorization).some(startsWith('AWS4-HMAC-SHA256'))) {
       return true
     }
```

Comment:
unrelated to the PR but what the hell is that ?

Reply:
Propagating headers with AWS was feature-gated as, if I recall correctly, there was some issue when signatures are present. @rochdev probably remembers better why we are doing this.